openapi.yml

```diff
@@ -127,6 +127,10 @@ paths:
       responses:
         '200':
           description: OK
+          content:
```

Comment:
Can you also add the Location header as well?

Reply:
How does this look?

Reply:
Love it.